Warmup-1/intMax/src/index.ts

```diff
@@ -0,0 +1,12 @@
+export function intMax(a: number, b: number, c: number): number {
+  //   return Math.max(a, b, c);
```

Comment:
Why is this commented out?

Reply:
It is a solution, but I commented it out for the sake of practicing loops.

Reply:
Fair, but in the future rather create a commit with it and then commit on top of it with a different solution. That way it's in history. Comments are not necessarily good practice